### ISSUE TITLE
Doesn't save any images to disk

### ISSUE THREAD

**Issue by n00mkrad** [lucidrains/deep-daze#62]
The current version (pip) doesn't save any images, no matter which save_every value.

Tested locally (Win 10) and on the simplified Colab notebook.

**Comment by lucidrains** [comment]
@n00mkrad ohh, for some reason it is hidden behind a CLI flag now `--save-progress`, but i defaulted it to true in the latest version! https://github.com/lucidrains/deep-daze/releases/tag/0.6.3

**Comment by afiaka87** [comment]
Nice I think that was my bad. I always use save-progress.

**Comment by n00mkrad** [comment]
I mean the CLI can't display images, I don't see the point of it being disabled by default lol

**Comment by lucidrains** [comment]
@n00mkrad I think @afiaka87 wanted to only save it at the very end

**Comment by afiaka87** [comment]
Yeah I believe that's how it turned out.  Ultimately just not testing with the CLI (I'm using the notebook) as I dont have a great GPU locally and not testing all the parameters before shipping. My bad.
